sdk/tests/wallet/balance.rs

```diff
@@ -142,7 +141,8 @@ async fn balance_expiration() -> Result<()> {
     assert_eq!(balance.base_coin().available(), 0);
 
     // Wait until expired
-    tokio::time::sleep(std::time::Duration::from_secs(seconds_until_expired.into())).await;
+    // TODO wait for slots, not seconds
+    tokio::time::sleep(std::time::Duration::from_secs(slots_until_expired)).await;
```

Comment:
Should be pretty easy to do right now

Reply:
Will do later still, need this merged ASAP